chart/compass/charts/pairing-adapter/values.yaml

```diff
@@ -1,18 +1,21 @@
 deployment:
   replicaCount: 1
   port: 8080
-  oauthSecret: "pairing-adapter-oauth"
   image:
     pullPolicy: IfNotPresent
   resources: {}
   securityContext: # Set on container level
     runAsUser: 2000
     allowPrivilegeEscalation: false
-  envs:
-    mappingTemplateExternalURL: ""
-    mappingTemplateHeaders: ""
-    mappingTemplateJSONBody: ""
-    mappingTemplateTokenFromResponse: ""
-    oauthStyle: "AuthDetect"
   strategy: {} # Read more: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
   nodeSelector: {}
+  adapters:
+    adapter0:
```

Comment:
`example` also seems applicable to me.
Maybe rename the tenant fetcher job as well (it seems like it uses the same declaration principle) - either to `example`, or `job0`